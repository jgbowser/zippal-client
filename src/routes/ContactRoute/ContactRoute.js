import React, {Component} from 'react'
import ContactBox from '../../components/ContactBox/ContactBox'


export default class ContactPage extends Component {
  render(){
    return(
        <div className='contact'>
          <h2>Contact Us</h2>
          <br/>
          <h3>Team Zip Pal</h3>
          <ContactBox />
        </div>
    )
  }
}