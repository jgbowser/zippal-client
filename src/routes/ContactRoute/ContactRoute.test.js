import React from 'react'
import ReactDOM from 'react-dom'
import ContactRoute from './ContactRoute'
import renderer from 'react-test-renderer'

describe.only('ContactRoute Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ContactRoute />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
  it(`takes snapshot of ContactRoute component`,()=>{
    const tree = renderer
    .create(<ContactRoute/>)
    expect(tree).toMatchSnapshot()
  })
})