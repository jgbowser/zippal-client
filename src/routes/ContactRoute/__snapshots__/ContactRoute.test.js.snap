// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactRoute Component takes snapshot of ContactRoute component 1`] = `
<div
  className="contact"
>
  <h2>
    Contact Us
  </h2>
  <br />
  <h3>
    Team Zip Pal
  </h3>
  <div
    className="contact_card_box"
  >
    <button
      className="contact_card"
      onClick={[Function]}
    >
      <h3>
        John
      </h3>
    </button>
    <button
      className="contact_card"
      onClick={[Function]}
    >
      <h3>
        Phillip
      </h3>
    </button>
    <button
      className="contact_card"
      onClick={[Function]}
    >
      <h3>
        Mathew
      </h3>
    </button>
    <button
      className="contact_card"
      onClick={[Function]}
    >
      <h3>
        Ryan
      </h3>
    </button>
  </div>
  
</div>
`;
